app/src/main/java/org/dslul/openboard/inputmethod/latin/setup/SetupWizardActivity.java

```diff
@@ -434,8 +435,8 @@ public SetupStep(final int stepNo, final String applicationName, final TextView
                 final int paddingEnd = mActionLabel.getPaddingEnd();
                 mActionLabel.setPaddingRelative(paddingEnd, 0, paddingEnd, 0);
             } else {
-                mActionLabel.setCompoundDrawablesRelativeWithIntrinsicBounds(
-                        actionIconDrawable, null, null, null);
+                actionIconDrawable.setBounds(0,0,60,60); // 60 appears to be equivalent to 24dp width and height
```

Comment:
dp  -> px conversion depends on the screen (on my S4 Mini, 24 dp is 36 pixels).
You can get the correct size using `dp * getResources().getDisplayMetrics().density`

Reply:
Thanks. I changed it to use the correct size.